src/components/pages/order/Main/admin/PanelTabs.js

```diff
@@ -0,0 +1,73 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import Tab from "../../../../reusable-ui/Tab";
+import { FiChevronDown } from "react-icons/fi";
+import { FiChevronUp } from "react-icons/fi";
+import { theme } from "../../../../../theme";
+import OrderContext from "../../../../../context/OrderContext";
+import { tabsConfig } from "./tabsConfig";
+
+export default function PanelButtons() {
```

Comment:
```suggestion
export default function PanelTabs() {
```

Conserver le même nom de composant et nom de fichier